src/melodyMaker.js

```diff
@@ -20,13 +33,8 @@ function createMelody() {
         notesArray.push([pickValidRandomNote(validNotes), time, time + thisNoteLength])
         time += thisNoteLength
     }
-    validNotes.forEach((note) => {
-        notesArray.push([noteFrequencyMap[note][2], time, time + 1])
-        time += 1
-    })
 
-    notesArray.push([noteFrequencyMap[validNotes[0]][2], time, time + 3])
-    var melodyParameters = [notesArray, tonic[0], scaleNames]
+    var melodyParameters = [notesArray, tonic[0], scaleNames, createScale(tonic, scale)]
```

Comment:
if you return an object here you dont have to know what index to lookup on the other side. instead you can look up by key name

Reply:
great call, I'll get this in a fast-follow